routers/web/repo/commit.go

```diff
@@ -29,6 +29,7 @@ import (
 	"code.gitea.io/gitea/services/context"
 	"code.gitea.io/gitea/services/gitdiff"
 	git_service "code.gitea.io/gitea/services/repository"
```

Comment:
Related to the CI failure
Why it is called `git_service`?

Reply:
Seems like typo and it should be named `repo_service`.

Reply:
Fixed